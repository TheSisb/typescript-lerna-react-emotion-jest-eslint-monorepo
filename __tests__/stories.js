import {resolve} from 'path';
import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({configPath: resolve(__dirname, '../.storybook')});
