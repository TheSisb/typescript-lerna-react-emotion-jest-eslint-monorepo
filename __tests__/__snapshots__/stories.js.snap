// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Heading Default 1`] = `
<span>
  I am a Very Large Heading
</span>
`;

exports[`Storyshots Text Default 1`] = `
<span>
  Some text
</span>
`;
